### ISSUE TITLE
Why does `brew.sh` require sudo?

### ISSUE THREAD

**Issue by jonnybarnes** [mathiasbynens/dotfiles#647]
The only line I can see that uses sudo is:
```
sudo ln -s /usr/local/bin/gsha256sum /usr/local/bin/sha256sum
```
But installing brew should make `/usr/local` user-writable.

**Comment by jrahmy** [comment]
I think the larger point was is there anything in that file that requires sudo then? The first couple of lines are a prompt/keep-alive for a sudo session.

**Comment by mathiasbynens** [comment]
@jrahmy Good point. Fixed in 5689fa8c011b1a5428a82a981737a5e3878a0cd1!
